style.css

```diff
@@ -5,3 +5,87 @@
   font-family: 'Kalam', cursive;
```

Comment:
The project looks good but README should be professional. Please add a project title, description, and instructions to set the project up locally. The live demo link is broken. Please add an active link or remove it. Please note that README should be clean and professional with only relevant details.